drape_frontend/drape_engine.cpp

```diff
@@ -55,9 +55,9 @@ DrapeEngine::DrapeEngine(Params && params)
 
   std::vector<PostprocessRenderer::Effect> effects;
 
-  bool enabledAntialiasing;
-  if (!settings::Get(dp::kSupportedAntialiasing, enabledAntialiasing))
-    enabledAntialiasing = false;
+//  bool enabledAntialiasing;
+//  if (!settings::Get(dp::kSupportedAntialiasing, enabledAntialiasing))
+//    enabledAntialiasing = false;
 
```

Comment:
So according to the code below, we always work without AA? effects vector is empty ..

Reply:
Yes